aiomysql/connection.py

```diff
@@ -596,7 +600,7 @@ def write_packet(self, payload):
     async def _read_bytes(self, num_bytes):
         try:
             data = await self._reader.readexactly(num_bytes)
-        except asyncio.streams.IncompleteReadError as e:
+        except IncompleteReadError as e:
```

Comment:
From the docs, it looks like it should be
```suggestion
        except asyncio.LimitOverrunError as e:
```
and no imports needed.

Reply:
@dongweiming pls

Reply:
@webknjaz @Dazix fixed  👍